index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Ivan Martinez - Portfolio</title>
+   <link rel="stylesheet" href="css/styles.css">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+   <section class="sec-portada">
+      <header> 
+        <a href="./index.html" class="logo">My Logo</a>
+         <nav>
+            <i id="id-menu-hamburger" class="menu-hamburg fa fa-bars"></i>
+            <ul class="menu">
+               <li class="menu-option">Hello</li>
+               <li class="menu-option">Portfolio</li>
+               <li class="menu-option">About</li>
+               <li class="menu-option">Contact</li>
+            </ul>
+         </nav>
+      </header>
+      
+      <section id="port-without-header">
+         <div class="port-contenido">
+            <h1 id="tit-port" class="title">I’m Ivan Glad to see you!</h1>   
+            <p id="intro-text" class="parrafo">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>   
+            <a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connect-text">LET'S CONNECT</a>
```

Comment:
- [ ] Kindly make sure the let's connect is not wrapped in a `anchor` tag as you have your social media links on the list items. `hint: You may consider using a header tag hx (examples: <h3>,<h4>,<h5>, etc...)` for semantic purpose